src/app/screens/Receive/index.tsx

```diff
@@ -232,47 +238,196 @@ function Receive() {
           />
         }
       />
-      {invoice ? (
-        <Container maxWidth="sm">{renderInvoice()}</Container>
+
+      {isAlbyUser ? (
+        <Container maxWidth="sm">
+          <div className=" h-full mx-auto">
+            <Tab.Group>
+              <Tab.List className="flex mb-6 mt-4 items-center  px-4">
+                <TabComponent
+                  active={recieveType === "lightning"}
+                  onClick={() => {
+                    setRecieveType("lightning");
+                  }}
+                  leftTab={true}
+                  icon={
+                    <svg
+                      width="16"
+                      height="18"
+                      viewBox="0 0 16 18"
+                      className={
+                        recieveType === "lightning"
+                          ? " text-gray-700 dark:text-neutral-200"
+                          : " text-gray-500 dark:text-neutral-500 hover:text-gray-700 "
+                      }
+                      xmlns="http://www.w3.org/2000/svg"
+                    >
+                      <path
+                        d="M14.4961 7.70883L5.8603 16.5885C5.6207 16.8349 5.22228 16.5503 5.37764 16.2437L8.45182 10.1779C8.55296 9.97833 8.40795 9.74227 8.18423 9.74227H1.71762C1.45129 9.74227 1.31702 9.42108 1.5041 9.23153L9.51318 1.11699C9.74554 0.881572 10.132 1.14034 10.0029 1.44487L7.70601 6.86069C7.62148 7.06 7.76944 7.28049 7.98592 7.2778L14.2773 7.19969C14.5444 7.19638 14.6823 7.51738 14.4961 7.70883Z"
+                        stroke="#374151"
+                      />
+                    </svg>
+                  }
+                  label="Lightning"
+                ></TabComponent>
+
+                <TabComponent
+                  active={recieveType === "onchain"}
+                  onClick={() => {
+                    setRecieveType("onchain");
+                  }}
+                  rightTab={true}
+                  icon={
+                    <svg
```

Comment:
I don't think the svg code should be directly embedded. Can you add them as a file to static/assets/icons?